src/Command/ExportToCsvCommand.php

```diff
@@ -66,28 +66,29 @@ protected function getDataFromRepository($input): array
             ? $this->repository->findBy(['id' => $input->getArgument(self::ITEM_IDS)])
             : $this->repository->findAll();
     }
-    protected function saveDataToCsv(OutputInterface $output, array $getDataFromRepository, $handler, float $timeStart): void
+    protected function saveDataToCsv(InputInterface $input, OutputInterface $output, array $getDataFromRepository, $handler, float $timeStart): void
     {
+        $data = $this->serializer->normalize($getDataFromRepository, 'csv', ['attributes' => $this->attributes]);
+        fwrite($handler, $this->serializer->serialize($data, 'csv'));
+        fclose($handler);
         if (!$getDataFromRepository = $this->getDataFromRepository($input->getArgument(self::ITEM_IDS))) {
```

Comment:
Po co drugi raz pobierasz ids? Jeśli raz getDataFromRepository zwrociło pusta tablicę, wywołane po raz drugi zrobi to samo